packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -172,7 +218,7 @@ const CancelSubscriptionPanel = ({
             <div className="button-row">
               <button
                 className="settings-button primary-button"
-                onClick={hideCancel}
+                onClick={engagedOnClick}
```

Comment:
I think `hideCancel` is more readable here. Could `engagedOnClick` be called from inside `hideCancel`? (OK if not)